aspnetcore/blazor/performance/event-pipe-profiling.md

```diff
@@ -0,0 +1,119 @@
+---
+title: ASP.NET Core Blazor WebAssembly Event Pipe performance profiling and diagnostic counters
+author: guardrex
+description: Learn about Event Pipe performance profiling and diagnostic counters in ASP.NET Core Blazor WebAssembly apps.
+monikerRange: '>= aspnetcore-10.0'
+ms.author: riande
+ms.custom: mvc
+ms.date: 04/29/2025
+uid: blazor/performance/event-pipe-profiling
+---
+# ASP.NET Core Blazor WebAssembly Event Pipe performance profiling and diagnostic counters
```

Comment:
"Event Pipe" can also give you GC heap dump. It's a major feature.
But I'm not sure we need all  features in the title.

Perhaps simpler "ASP.NET Core Blazor WebAssembly diagnostic with Event Pipe"